### PR TITLE
Remove keystone-operator from infra-operator kuttl tests

### PR DESCRIPTION
**Pull request by kajinamit**
infra-operator no longer depends on keystone-operator because we removed OpenStackClient CR from infra-operator.

### CODE REVIEW

**Comment by kajinamit** [comment]
This is related to https://github.com/openstack-k8s-operators/infra-operator/pull/90 which removed OpenStackClient CR from infra-operator

**Comment by kajinamit** [comment]
```
"fatal: [localhost]: FAILED! => {\"changed\": true, \"cmd\": \"oc rsh -n openstack openstackclient openstack compute service list;\\noc rsh -n openstack openstackclient openstack network agent list;\\n\", \"delta\": \"0:00:00.223159\", \"end\": \"2023-07-19 22:20:06.710291\", \"msg\": \"non-zero return code\", \"rc\": 1, \"start\": \"2023-07-19 22:20:06.487132\", \"stderr\": \"Error from server (NotFound): pods \\\"openstackclient\\\" not found\\nError from server (NotFound): pods \\\"openstackclient\\\" not found\", \"stderr_lines\": [\"Error from server (NotFound): pods \\\"openstackclient\\\" not found\", \"Error from server (NotFound): pods \\\"openstackclient\\\" not found\"], \"stdout\": \"\", \"stdout_lines\": []}",
2023-07-19 22:20:07.211595 | controller |         "",
```

We have to force-merge this or probably wait until the openstack-operator PR is merged.

**Review by cescgina — COMMENTED** [review]
tested in crc, works well, lgtm

**Comment by kajinamit** [comment]
recheck

**Comment by abays** [comment]
recheck

**Comment by abays** [comment]
recheck

**Comment by kajinamit** [comment]
recheck

**Comment by kajinamit** [comment]
recheck

**Comment by kajinamit** [comment]
recheck

**Comment by kajinamit** [comment]
recheck

**Comment by abays** [comment]
recheck

**Comment by abays** [comment]
recheck

**Comment by abays** [comment]
Tests have passed independently across the various batched runs. Will override.

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**
